src/components/plots/piePlot.ts

```diff
@@ -2,6 +2,12 @@
 
 module Plottable {
 export module Plots {
+
+  export interface PieValueBinding<D> {
```

Comment:
Can you use whatever the generalized name will be for this, so we can see what the final API might look like on other `Plot`s?